data/frameSimulator/src/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+
+set(CMAKE_INCLUDE_CURRENT_DIR on)
+ADD_DEFINITIONS(-DBOOST_TEST_DYN_LINK)
+
+include_directories(${FRAMESIMULATOR_DIR}/include ${ODINDATA_INCLUDE_DIRS}
+	${Boost_INCLUDE_DIRS} ${LOG4CXX_INCLUDE_DIRS}/.. ${ZEROMQ_INCLUDE_DIRS} ${PCAP_INCLUDE_DIR})
```

Comment:
Nitpicking but: should "Boost_INCLUDE_DIRS" be all capitals i.e. "BOOST_INCLUDE_DIRS"?
I suspect that CMake is case insensitive though.